src/test/index.ts

```
@@ -110,7 +110,7 @@ function activatePythonExtensionScript() {
     const initializationPromise = initialize();
     const promise = Promise.race([initializationPromise, failed]);
     // tslint:disable-next-line: no-console
-    promise.catch(ex => console.error(ex)).finally(() => clearTimeout(timer!));
+    promise.catch(e => console.error(e)).finally(() => clearTimeout(timer!)).ignoreErrors();
```

`promise.finally(() => clearTimeout(timer!)).catch(e => console.error(e));`

I believe this is better. as we dont want to ignore errors.

---

tslint is giving me a warning saying no floating promises, which is why I did this.

---

Yes, if you reverse the code then it will be fine.
I.e. you need to ensure `catch` is the last thing. With the suggestion I have provided it should work.